tls/tls.md

```diff
@@ -42,16 +42,25 @@ The libp2p handshake uses TLS 1.3 (and higher). Endpoints MUST NOT negotiate low
 
 During the handshake, peers authenticate each other’s identity as described in [Peer Authentication](#peer-authentication). Endpoints MUST verify the peer's identity. Specifically, this means that servers MUST require client authentication during the TLS handshake, and MUST abort a connection attempt if the client fails to provide the requested authentication information.
 
+When negotiating the usage of this handshake dynamically, via a protocol agreement mechanism like [multistream-select 1.0](https://github.com/libp2p/specs/blob/master/connections/README.md#multistream-select), it MUST be identified with the following protocol ID:
```

Comment:
Better: it is identified. We're just giving it a name here, no need for normative language.

Reply:
Needs to be normative for other implementations to comply, unless I'm missing something?

Reply:
I don't think the assignment of a code point requires normative language. But keep it that way if you prefer.